admin_ui_support/modules/admin_ui_widget_example/js/src/options_buttons.widget.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import InputLabel from '@material-ui/core/InputLabel';
+import FormControl from '@material-ui/core/FormControl';
+import FormGroup from '@material-ui/core/FormGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Checkbox from '@material-ui/core/Checkbox';
+import FormHelperText from '@material-ui/core/FormHelperText';
+
+const OptionsButtons = ({
+  label,
```

Comment:
We should also document available props in widgets somewhere

Reply:
I am wondering whether the storybook could be a place to document something like this? It could contain examples as well...

Reply:
As I mentioned on the other thread, I think storybook is a good place to document this for now so let's do that 👍